recipe/meta.yaml

```diff
@@ -49,15 +42,18 @@ test:
     - if not exist %LIBRARY_INC%\\mpi.f90 exit 1  # [win]
     - if not exist %LIBRARY_INC%\\mpi.h exit 1  # [win]
     - if not exist %LIBRARY_LIB%\\msmpi.lib exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\\mpifort.lib exit 1  # [win]
     - if not %MSMPI_BIN% == %LIBRARY_BIN% exit 1  # [win]
     - if not %MSMPI_INC% == %LIBRARY_INC% exit 1  # [win]
     - if not %MSMPI_LIB64% == %LIBRARY_LIB% exit 1  # [win]
 
     # check if binaries work ok
     - where mpiexec.exe  # [win]
     - where smpd.exe  # [win]
-    - mpiexec.exe -d 3 -n 2 Tests\\MpiPingpong.exe -m  # [win]
-    - mpiexec.exe -d 3 -n 2 Tests\\IMB-MPI1.exe PingPong PingPing PingPongAnySource PingPingAnySource Sendrecv Allreduce -off_cache -1 -msglog 10  # [win]
+    
```

Comment:
Could you kill this old stuff, and maybe replace it with:

```
mpiexec /help2
mpiexec /help3
```
https://github.com/mpi4py/setup-mpi/blob/master/.github/workflows/ci.yml#L84